README.md

```diff
@@ -18,9 +18,43 @@
 ## Usage
 
 1. Choose your flavour.
-2. Copy the contents of _flavour_.cava into your Cava config file (usually located at: `$HOME/.config/cava/`), replacing the existing gradient settings.
+2. Copy the contents of _flavour_.cava into your Cava config file, as specified in [Cava's README](https://github.com/karlstav/cava#configuration). Typically, the config file is located at: `~/.config/cava/`. Ensure you replace the existing gradient settings.
```

Comment:
According to [the Cava docs](https://github.com/karlstav/cava?tab=readme-ov-file#configuration), the file is located at `~/.config/cava/config`, rather than `~/.config/cava`. We should be specific.